### DISCUSSION TITLE
FnB TimeSeries - Error pada saat melakukan evaluasi

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#317] [Q&A] [answered]
Halo, saya menemukan error pada saat ingin melakukan evaluasi terhadap model saya, berikut adalah contoh error nya: 
![Error1](https://user-images.githubusercontent.com/107049922/191483756-145b0e03-020c-437c-8fc6-d999ec1409bc.png)

Berikut adalah model yang telah saya buat: 
![Error2](https://user-images.githubusercontent.com/107049922/191484775-2b801ca7-5df2-492c-971e-0badd68a60aa.png)

Berikut adalah data validation yang saya miliki:
![Error3](https://user-images.githubusercontent.com/107049922/191483920-964c3060-6ed4-43f9-84a3-bbd3d3e2b1f8.png)

Mohon bantuannya yaa, terima kasih :)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @RanggaGemilang ,

Terima kasih telah melampirkan error beserta kode yang digunakan sehingga membantu kami untuk mengidentifikasikan masalah yang Anda alami. 

Selain memastikan `fnb_val` adalah object time series, mohon dipastikan juga object `f_arima$mean` adalah object time series dengan panjang yang sama dengan `fnb_val`. Ketika hal tersebut sudah benar, mungkin permasalahan adalah saat pemanggilan fungsi `accuracy()` yang bisa saja mengacu untuk package lain. Karena kita ingin menggunakan `accuracy()` dari package `forecast` maka dari itu Bapak dapat mencoba kode di bawah ini:
`forecast::accuracy(f_arima$mean, fnb_val)`

Jika masih terdapat error silahkan balas pesan ini sehingga kami dapat bantu. Jika jawaban ini dirasa membantu silahkan untuk menandai jawaban ini "Mark as Answer".

Sekian dan terima kasih

**Reply by RanggaGemilang** [reply]
Terima kasih banyak mas @IrfanChairurrachman, ternyata memang fungsi accuracy() nya mengacu kepada package lain dan sudah bisa dijalankan dengan menambahkan forecast::accuracy()
